### ISSUE TITLE
[ADE Support]: Accredited Representation Management: Representative Status in Profile: 508 Audit Report Findings - Defect 6

### ISSUE THREAD

**Issue by lakper** [department-of-veterans-affairs/va.gov-team#112275]
> [!NOTE]
> **QUICK SUMMARY:** _Child ticket of https://github.com/department-of-veterans-affairs/va.gov-team/issues/112265 tracking defect 6._

### Defect 6 | Critical | Created:
**_WCAG 4.1.2 Name, Role, Value | For all user interface components (including but not limited to: form elements, links and components generated by scripts), the name and role can be programmatically determined; states, properties, and values that can be set by the user can be programmatically set; and notification of changes to these items is available to user agents, including assistive technologies._**

On the Get Help From A VA Accredited Representative Or VSO | Veterans Affairs (No Representative) screen (Get help from an accredited representative or VSO), under Option 2: Submit your form by mail, the name of the button cannot be understood in relation to the state. For example, “Expand all accordions Button collapsed” is read. One possibility is to change the name, “Expand all accordions” or “Collapse all accordions”, without indicating the state. Another possibility is to keep the name the same, “all accordions Button collapsed” or “all accordions Button expanded”, and change the state—this is the convention used throughout the product. A similar example can be found on the VA Accredited Representative FAQs | Veterans Affairs (Represented by an attorney or claims agent) screen (Learn about accredited representatives). (New: 06/16/2025)

## Supporting Artifacts 
- [http://staging.va.gov/profile/accredited-representative](http://staging.va.gov/profile/accredited-representative)

### Ticket Updates
- [ ] _Connect to an `Epic/Parent` or `Intake` (what body of work is this a part of?) If this is the `Epic/Parent`, connect related tickets as sub-issues._
- [ ] _If this is the `Epic/Parent` ticket, add label for `ADE 508 RECHECK`_
- [ ] _If this ticket is a `Child` ticket aka `sub-issue` please add the appropriate labels for `ADE 508 FLAG CRITICAL`;`ADE 508 FLAG HIGH`; `ADE 508 FLAG MEDIUM` or `ADE 508 FLAG LOW`_
- [ ] _Label with `Originator/Team` (product team or stakeholder requesting support; ADE if internal)_
- [ ] _Label date in the `Open Date` field_
- [ ] _Label with `Estimate` (level of effort expected for this ticket)_
- [ ] _Add `Assignee(s)` name(s) to ticket_ (include Lakwi)
- [ ] _Add `Assignee(s) name(s) to each task` they will complete via handle tag (if known)_
- [ ] _Select a `Priority Level`_
- [ ] _Update date in `Last Checked` field_
- [ ] _Label with `Actual` (level of effort it took to complete this ticket)_
- [ ] _Update date in_ `Closed Date`
- [ ] _Make sure the `ADE 508 RECHECK` label is on this ticket upon closing and that a follow-up date has been established and calendared._
